.github/workflows/release.yaml

```diff
@@ -59,8 +54,7 @@ jobs:
           tags: ${{ steps.meta-kusk-gateway.outputs.tags }}
           labels: ${{ steps.meta-kusk-gateway.outputs.labels }}
           build-args: TELEMETRY_TOKEN=${{ secrets.TELEMETRY_TOKEN }}
```

Comment:
Did you intentionally leave this here as you removed it elsewhere?

Reply:
Nope I missed it

Reply:
Does this still need to be removed?